ftw/participation/browser/participants.py

```diff
@@ -93,6 +100,24 @@ def __call__(self):
 
         return self.template()
 
+    def resend_invitations(self, invitation_uids):
+        for uid in invitation_uids:
+            self.storage.get_invitation_by_iid(uid)
+            all_invitations = self.storage._invitations_list
+            invite_form = InviteForm(self.context, self.request)
+
+            for email, user_invitations in all_invitations.items():
+                for invite in user_invitations:
```

Comment:
You could use the built-in filter here to avoid getting the tree structure here. But I'm not sure if it will make the code better readable.
https://book.pythontips.com/en/latest/map_filter.html#filter